config/s3/config.go

```diff
@@ -32,7 +32,9 @@ func Configure(p *config.Provider) {
 	})
 
 	p.AddResourceConfigurator("aws_s3_bucket_acl", func(r *config.Resource) {
-		config.MoveToStatus(r.TerraformResource, "acl")
```

Comment:
Hi @ytsarev,
My understanding is that this is another case where we previously removed a field from the API because there's another one that we though does the same job (e.g., in this case I believe it's the `spec.forProvider. accessControlPolicy`) but it turns out that the remaining API is not sufficient to handle certain use cases (in this specific case, being able to define canned ACLs). Is this correct?

Reply:
@ulucinar that's correct,[canned ACLs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#canned-acl) are simple strings, aka prepackaged ACLs by the cloud provider. `accessControlPolicy` is a full-fledged ACL doc for the granular control. We should not force users to use a more complex `accessControlPolicy` when it is actually not required